terraform/main.tf

```diff
@@ -0,0 +1,44 @@
+terraform {
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = "3.49.0"
+    }
+  }
+
+  backend "s3" {
+    key     = "terraform.tfstate"
+    encrypt = true
+  }
+}
+
+provider "aws" {
+  region = var.region
+  default_tags {
+    tags = {
+      Environment = var.target_env
+      Project     = var.project_code
+      Author      = "FreshWorks Studio"
+    }
+  }
+}
+
+provider "aws" {
+  alias  = "us-east-1"
+  region = "us-east-1"
```

Comment:
Shouldn't this be ca-central-1?

Reply:
we need two providers - for deploying edge lambda - Cloudfront functions

Reply:
one for ca-central - as above, this one for edge lambda